### ISSUE TITLE
Aria role usage

### ISSUE THREAD

**Issue by MikeCrockerEIS** [emailmonday/Cerberus#117]
I believe aria-hidden="true" on table tags will also hide descendants, which I suspect is not intended in your implementation. I use aria-hidden="true" only where the content is truly for presentation.

Would you agree with this? It is very possible I am misinformed.

Thanks

Mike

**Comment by TedGoas** [comment]
I did some digging and you're on to something! I asked a few email accessibility folks, screenshot of conversation below.

**TL;DR:** 
* `aria-hidden="true"` will not read out the element or any content. It should be used sparingly to hide things like spacers and decorative images.
* `role="presentation"` will not read out the element type and will read the content as if there were not wrapper (or like wrapped in div or span). Every `<table>` except data tables should include this so screen readers know they're for layout only.

Seems I went a little heavy handed with `aria-hidden="true"`! I can work through the templates and remove instances that don't make sense. Thanks for pointing this out!

___

![aria slack convo](https://user-images.githubusercontent.com/1172461/27395261-3262d6b4-567d-11e7-83d6-cd093a17fbe0.png)

**Comment by MikeCrockerEIS** [comment]
Glad to be of help, thanks for the awesome templates!
